packages/js/client/README.md

```diff
@@ -45,11 +47,21 @@ const config = {
       to: "wrap://ens/to.eth",
     }
   ],
-  // declare and configure plugin wrappers
-  plugins: [
+  // add embedded Wasm wrappers
```

Comment:
in line `43` of this file, there's `// redirect queries from one uri to another` - I would recommend to change it to `// redirect invocation from one uri to another`

it doesn't allow me to add the comment in that line because it wasn't modified lol